CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+
+I will try to help and take every contribution seriously.
+
+This is a great opportunity for me to learn how you use the role and also an opportunity for you to get into the habit of contributing to open-source software.
+
+## [Step by step](#step-by-step)
+
+Here is how you can help, many steps are related to GitHub, not specifically to this project.
+
+### [1. Make an issue.](#1-make-an-issue)
+
+When you spot an issue, [create an issue](https://github.com/vitabaks/postgresql_cluster/issues).
+
+Making the issue helps me and others to find similar problems in the future.
+
+### [2. Fork the project.](#2-fork-the-project)
+
+On the top right side of [the repository on GitHub](https://github.com/vitabaks/postgresql_cluster), click `fork`. This copies everything to your GitHub namespace.
+
+### [3. Make the changes](#3-make-the-changes)
+
+In your GitHub namespace, make the required changes.
+
+I typically do that by cloning the repository (in your namespace) locally:
+
+```
+git clone git@github.com:YOURNAMESPACE/postgresql_cluster.git
+```
+
+Now you can start editing on your laptop.
+
+### [4. Test your changes](#4-optionally-test-your-changes)
+
+
+#### 4.1 Contributing with Gitpod
+
+Gitpod is a cloud-based development environment platform that allows you to contribute to projects without setting up a local environment. You can use Gitpod to work on the `postgresql_cluster` project by following these steps:
+
+1. **Sign up for Gitpod**: If you don't have a Gitpod account, sign up at https://gitpod.io using your GitHub account.
+
+2. **Create a new fork**: If you haven't already, fork the `postgresql_cluster` repository by clicking the "Fork" button on the top-right corner of the repository's GitHub page (https://github.com/vitabaks/postgresql_cluster).
+
+3. **Open your fork in Gitpod**: Replace the `username` in the following URL with your GitHub username and open it in a new browser tab: `https://gitpod.io/#https://github.com/username/postgresql_cluster`. This will launch a new Gitpod workspace with the `postgresql_cluster` project.
+
+4. **Create a new branch**: In the Gitpod terminal, create a new branch for your changes:
+
+   ```
+   git checkout -b my-feature-branch
+   ```
+
+   Replace `my-feature-branch` with a descriptive name for your branch.
+
+5. **Make your changes**: Modify the code, add new features, or fix existing issues in the project. Make sure to follow the project's coding guidelines and style.
+
+6. **Commit your changes**: Once you've made your changes, stage and commit them using the following commands:
+
+   ```
+   git add .
+   git commit -m "Add a brief description of your changes"
+   ```
+
+   Replace the commit message with a clear and concise description of your changes.
+
+7. **Push your changes**: Push your changes to your fork on GitHub:
```

Comment:
It seems that the "Test your changes" item is missing here before pushing changes

Reply:
good. If you have time, please provide a PR for gitpod based on how you see it.

But I think it is important that gitpod is already fully configured when starting the production environment.

It is necessary to save time, because it is the most valuable resource)

Reply:
Done and tested on gitpod (working)